cli/flags.rs

```diff
@@ -1672,6 +1678,12 @@ Only local files from entry point module graph are watched.",
   }
 }
 
+fn no_clear_screen_arg<'a>() -> Arg<'a> {
+  Arg::new("no-clear-screen")
```

Comment:
This arg should require "--unstable" flag to be present

Reply:
@ah-yu sorry for the noise; I meant the `--watch` flag, not the `--unstable` flag...

Reply:
@bartlomieju Fixed it.